templates/patterns/content_banner/content-banner.ui_patterns.yml

```diff
@@ -0,0 +1,41 @@
+content_banner:
+    label: 'Content banner'
+    description: 'Shows the content summary along with the title, image and badges.'
+    fields:
+      background:
+        type: string
+        label: 'Background'
+        description: 'Set a gray or white background at the content banner. Defaults to white.'
+        escape: false
+        preview: 'gray'
+      image:
+        type: ImageValueObject
```

Comment:
Technically, we also support an array of the format `array{src: string, alt: string}`.
Should we document this? Or is this a hidden feature we don't want people to use?

Reply:
No when we use ImageValueObject we should support only that.

Reply:
Problem is we (currently) can't create these value objects in twig when including from another template, and we can't create them in yml for test fixtures.

So there are different options:
- technically support plain arrays instead of value objects, for pattern input. This is kind of what we currently do, we technically support it but we don't document it.
- give up on yml test fixtures for these patterns, _and_ create the value objects in preprocess when we want to include the pattern in a template. This is what oe_theme does. Does not look ideal to me.
- Create twig functions to construct value objects. This still leaves the problem for yml test fixtures.
- Give up on value objects. Only applicable to new patterns, which this is.

I noticed in oe_theme we don't have yml fixtures for these patterns, and we always construct the value objects in preprocess, e.g. oe_theme_content_person_preprocess_oe_theme_helper_in_page_navigation__node__oe_person().

There are also some patterns where we don't use value objects.

Overall this makes me question the usefulness. But